.prettierrc

```
@@ -0,0 +1,5 @@
+{
+  "endOfLine": "auto",
+  "singleQuote": true,
+  "arrowParens": "avoid"
```

we need trailingComma to match sdk. it's also the default in Prettier v3 (to come)
```suggestion
  "trailingComma": "all",
  "arrowParens": "avoid"
```

also consider alphabetizing

---

Done